ext/mysql2/client.h

```diff
@@ -3,7 +3,7 @@
 
 typedef struct {
   VALUE encoding;
-  VALUE active_thread; /* rb_thread_current() or Qnil */
+  VALUE active_fiber; /* rb_thread_current() or Qnil */
```

Comment:
Minor nit: the comment should reflect `rb_fiber_current()`

Reply:
Thank you. Fixed.